plenum/server/consensus/ordering_service.py

```diff
@@ -1259,7 +1259,7 @@ def lastPrePrepareSeqNo(self, n):
         values else it will not. To forcefully override as in case of `revert`,
         directly set `self._lastPrePrepareSeqNo`
         """
-        if n > self._lastPrePrepareSeqNo:
+        if n > self._lastPrePrepareSeqNo or not self.is_master:
```

Comment:
What about a case of normal ordering on non-master where last_ordered is less than lastPrePrepareSeqNo?

Reply:
yes, i forgot this... sorry